### PR TITLE
blogpost: deploy FusionAuth on Digital Ocean

### PR DESCRIPTION
**Pull request by sixhobbits**

### CODE REVIEW

File: site/_posts/2023-03-30-digitalocean-oneclick-installation.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+{% include _image.liquid src="/assets/img/blogs/digitalocean/digitalocean-destroy-cluster.png" alt="Destroying a cluster from the DigitalOcean admin console" class="img-fluid" figure=false %}
+
+If you did not destroy the cluster, you can reinstall FusionAuth to this cluster by following the installation instructions above and selecting the current cluster instead of a new one.
+
+{% include _image.liquid src="/assets/img/blogs/digitalocean/digitalocean-select-current-cluster.png" alt="Selecting current cluster from the dropdown in the configuration wizard" class="img-fluid" figure=false %}
+
+
+## Next Steps
+
+You've now deployed a FusionAuth instance on a public Kubernetes cluster hosted on DigitalOcean. From here, you can [create an application](https://fusionauth.io/docs/v1/tech/core-concepts/applications) and [register users to it](https://fusionauth.io/docs/v1/tech/tutorials/register-user-login-api) to implement a login page. Be sure to check out the [guides](https://fusionauth.io/docs/v1/tech/guides/) for more information about how to implement FusionAuth's many features into your application.
```
Comment (ColinFrick) [inline site/_posts/2023-03-30-digitalocean-oneclick-installation.md:233]: Please use relative links

**Review by ColinFrick — CHANGES_REQUESTED** [review]
I had to re-create the cluster, because FusionAuth 1-click installation kept failing. I don't know what caused this. After destroying the cluster and creating a new one, it worked flawlessly.

**Comment by sixhobbits** [comment]
> I had to re-create the cluster, because FusionAuth 1-click installation kept failing. I don't know what caused this. After destroying the cluster and creating a new one, it worked flawlessly.

@ColinFrick - I meant to mention that we had this problem too but couldn't reliably reproduce it or figure out why it was happening. Not sure if it is something that the DigitalOcean team would need to look into or if we should raise a bug on Fusionauth?

**Comment by ColinFrick** [comment]
@mooreds the PR looks good to me, but it the deployment could fail sporadically (see https://github.com/FusionAuth/fusionauth-issues/issues/2297 ). How should we proceed?

**Comment by mooreds** [comment]
Looks like 2297 is going to be addressed soon (it is in the backlog) so we can wait until it is fixed before publishing.

File: site/_posts/2023-03-30-digitalocean-oneclick-installation.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+```sh
+Notice: Adding cluster credentials to kubeconfig file found in "/Users/<USERNAME>/.kube/config"
+Notice: Setting current-context to <YOUR_CLUSTER_NAME>
+```
+
+FusionAuth is now running on a publicly accessible IP address. To retrieve this address, run the following command in your terminal:
+
+```sh
+export SERVICE_IP=$(kubectl get svc --namespace fusionauth fusionauth -o jsonpath='{.status.loadBalancer.ingress[0].ip}') \
```
Comment (bhalsey) [inline site/_posts/2023-03-30-digitalocean-oneclick-installation.md:58]:
```suggestion
export SERVICE_IP=$(kubectl get svc --namespace fusionauth fusionauth -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
```

Remove backslash at the end of the first line (consistent with other places this snippet occurs).  Or insert a semicolon before the backslash `; \`

**Comment by mooreds** [comment]
Looks like we're good to go from a DO perspective.

**Comment by mooreds** [comment]
Closing this, will deliver under https://github.com/FusionAuth/fusionauth-site/pull/2493

**Pull request closed without merging**